src/mma7660/mma7660.h

```diff
@@ -30,7 +30,7 @@
 #include <mraa/i2c.h>
 #include <mraa/gpio.h>
 
-#include <mma7660_regs.h>
+#include "mma7660_regs.h"
```

Comment:
clang 3.9.0:

```
mma7660.h:33:10: error:
      'mma7660_regs.h' file not found with <angled> include; use "quotes" instead
#include <mma7660_regs.h>
         ^~~~~~~~~~~~~~~~
         "mma7660_regs.h"
```

GNU 5.2.1:

`mma7660.h:33:26: fatal error: mma7660_regs.h: No such file or directory
`

#include "blah" - local then system
#include <blah> - system
Of cource, the C Standard is a bit vague here and states that it's implementation specific.  

I imaging since we don't have the list from include_directories that we are now seeing some of these  - I fixed the failing cases (which may have been only C).